src/components/molecules/MessageList.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="bg-gray-infolt p-6 space-y-6 overflow-auto"> 
+        <message-card 
+            timestamp="Weds, May 14th, 10:04 am" 
+            title="Your Employment Insurance Claim - You've been subscribed to Job Alerts" 
+            paragraphs="Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum ea eos ducimus nulla sit quae, commodi eaque ut eveniet laudantium amet maxime consectetur, facere repellendus ipsum tenetur numquam officia mollitia?"    
+         />
+         <message-card 
+            timestamp="Weds, May 14th, 10:04 am" 
+            title="Your Employment Insurance Claim - You've been subscribed to Job Alerts" 
+            paragraphs="Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum ea eos ducimus nulla sit quae, commodi eaque ut eveniet laudantium amet maxime consectetur, facere repellendus ipsum tenetur numquam officia mollitia?"    
+         />
+         <message-card 
+            timestamp="Weds, May 14th, 10:04 am" 
+            title="Your Employment Insurance Claim - You've been subscribed to Job Alerts" 
+            paragraphs="Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum ea eos ducimus nulla sit quae, commodi eaque ut eveniet laudantium amet maxime consectetur, facere repellendus ipsum tenetur numquam officia mollitia?"    
+         />
+        <message-card />
+        <message-card title="Empty box above" />
+    </div>
+<!--
+<div class="flex flex-col h-96 bg-gray-light p-6 space-y-6 overflow-auto"> 
+    <div class="bg-white p-6 space-y-3" v:for="msg of messages">
+        <span class="uppercase text-sm font-light">{{ msg.timestamp }}</span>
+        <h1 class="text-2xl font-semibold">{{ msg.title }}</h1>
+        <p class="font-normal text-lg">{{ msg.paragraphs }}</p>
+    </div>
+</div> -->
+</template>
+
+<script>
+import MessageCard from "./MessageCard.vue"
+
+export default {
+    //  not 100% sure how to implemnet prop passing
+    //  props: {
+    //     messages: Array, //should be an array of objects with a timestamp, a header, and an array of paragraphs  
+    // },
```

Comment:
I agree with passing in an array of messages(objects) with the timestamp, header and text. We can call together and we can see how that is done. :)

i don't think the paragraphs need to be in an array. I believe when we create the data, we can simply add line breaks to the texts. We will see when that is done :p

Reply:
For now I've added css classes so that line breaks are preserved in the passed prop, but we can change it in the future if need be.